simulation/client.go

```diff
@@ -477,6 +498,27 @@ func (c *client) PostPorositySimulation(simulation *models.PorositySimulation) (
 	return response.Payload, nil
 }
 
+// GetSimulationOutputs get the outputs for a simulation
+func (c *client) GetSimulationOutputs(simulationID int32) (so []*models.SimulationOutput, err error) {
```

Comment:
Typically don't use `Get` in function names in Go.  I would just use `SimulationOutputs` for this.